pvacseq/bam-readcount.cwl

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "run bam-readcount"
+
+baseCommand: ["/usr/bin/perl", "helper.pl"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/bam-readcount-cwl:0.7.4"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'helper.pl'
+        entry: |
+            use warnings;
+            use strict;
+
+            use IO::File;
+            use File::Temp;
+
+            my $file = $ARGV[0];
+            my $sample = $ARGV[1];
+            my $bam_file = $ARGV[2];
+            my $ref_fasta = $ARGV[3];
+            my $snp_output = $ARGV[4];
+            my $indel_output = $ARGV[5];
+
+            my $input = IO::File->new($file);
+            my @vcf_header = parse_vcf_header($input);
+            my $header_line = $vcf_header[-1];
+            chomp $header_line;
+            my @header_fields = split \"\\t\", $header_line;
```

Comment:
I'm assuming the escapes are necessary as this is parsed as a string by bash? Otherwise, this would simply be `split "\t", $header_line;` right?